Source/AnimatableImageView.swift

```diff
@@ -1,19 +1,66 @@
 import UIKit
 
+/// A protocol which defines methods that allow you to manage the animation progress and respond to the animation state changes.
+@objc
+public protocol AnimatableImageViewDelegate: class {
```

Comment:
Ok. All changes regarding this protocol and delegate I'll submit later as a separate PR.